docs/modules/ROOT/pages/installation.adoc

```diff
@@ -57,11 +57,27 @@ docker run \
 
 == Building the operator from source
 
-This operator is written in Rust and is developed against the latest stable Rust release (1.58 at
+This operator is written in Rust and is developed against the latest stable Rust release (1.59 at
```

Comment:
I would get rid of the Rust version. This is a moving target that's not even relevant for users of the operator.